python/main.py

```diff
@@ -32,19 +32,30 @@ def dict_factory(cursor, row):
 
     return d
 
+def sha256_filename(filepath):
+    logger.info(f"Image path: {filepath}")
+
+    filepath = os.path.splitext(filepath)[0]
+    result = hashlib.sha256(filepath.encode("utf-8")).hexdigest()
+    return result
+
 @app.get("/")
 def root():
     return {"message": "Hello, world!"}
 
 @app.post("/items")
-def add_item(name: str = Form(...), category: str = Form(...)):
+def add_item(name: str = Form(...), category: str = Form(...), image: str = Form(...)):
     logger.info(f"Receive item: {name} in {category}")
 
-    item = (name, category)
+    #hash image file name by sha256
+    hash_img = sha256_filename(image) + '.jpg'
```

Comment:
I think maybe the task is not hashing the filename but the image itself?
這裡好像是要hash圖片本身(?

Reply:
It does make more sense to hash the image itself!
Seems like we misunderstood what the mentors meant by "sending the text" only.

Reply:
@tsoibet  It was my mistake that I didn't find this problem when I reviewed your code .

Reply:
@littlibanana No it's actually my fault. I didn't create any PR for this task. Didn't realize we are supposed to create PRs until task 8 in STEP3 :')

Reply:
@littlibanana @tsoibet Thank you for your comments and sorry for my late reply.
I've just seen the new messages in slack, so I think I don't have to replace the filename with the image, right?
If you have time, please help me approve the PR. Any other comments would be appreciated.

Reply:
Yes, but the command turns to be -F 'image=@images/local_image.jpg'. Thus,it is not a str but upload a .jpg file now.
def add_item(name: str = Form(...), category: str = Form(...), image: str = Form(...)): would cause some error that the input of image is not str but a file.
https://fastapi.tiangolo.com/tutorial/request-files/

Reply:
Thank you for your comments and information. I'll revise it.

Reply:
My understanding of the update on STEP3 is,
If you decide to take the challenge to implement image upload feature, then you will have to hash the image content and use the hash value as the new filename. 
Otherwise, just take the image path (a string), hash the filename and rename it.